packages/syntax-core/src/RichSelect/richSelectItems.ts

```diff
@@ -0,0 +1,11 @@
+import RichSelectSection from "./RichSelectSection";
+import RichSelectChip from "./RichSelectChip";
+import RichSelectRadioButton from "./RichSelectRadioButton";
+
+const richSelectItems = {
+  Section: RichSelectSection,
+  Chip: RichSelectChip,
+  RadioButton: RichSelectRadioButton,
+};
+
+export default richSelectItems;
```

Comment:
@benjroy nit could we avoid the re-export of the components? Feel free to import them directly in the files where they're being used.

Reply:
https://github.com/Cambly/syntax/pull/307/commits/e9ba5968526599fbb601aebf468a34a63e18d143